### PR TITLE
add my peer info page

### PR DESCRIPTION
**Pull request by HanaAyman**
This pull request is for WPC-483 and WPC-484. To make Edit peer info a separate page and make peering a dropdown navigation containing my peer info, peers and tokens. 
https://wepublish.atlassian.net/browse/WPC-484
https://wepublish.atlassian.net/browse/WPC-483 

Screenshot from 2021-09-27 16-55-43

### CODE REVIEW

File: packages/editor/src/client/routes/myPeerInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,224 @@
+import React, {useState, useEffect} from 'react'
+
+import {Button, Drawer, Form, FormGroup, ControlLabel, FormControl, Alert, FlexboxGrid} from 'rsuite'
+
```
Comment (penk85) [inline packages/editor/src/client/routes/myPeerInfo.tsx:4]: Maybe also delete the peer profile panel. We won't need that anymore.

**Review by penk85 — CHANGES_REQUESTED** [review]
Very nice change. Just have to be aware to merge the changes properly on the CTA image PR because it has additions on the peer profile panel (depending on merge order).

**Review by penk85 — APPROVED** [review]
:+1:

**Review by camilleory — APPROVED** [review]

File: packages/editor/src/client/routes/myPeerInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,224 @@
+import React, {useState, useEffect} from 'react'
```
Comment (bishoymelek) [inline packages/editor/src/client/routes/myPeerInfo.tsx:1]: @penk85 is this file needed? From where can you access this route?

Comment (penk85) [reply packages/editor/src/client/routes/myPeerInfo.tsx:1]: @bishoymelek you're right. This should go :+1:

**Review by lakeside763 — DISMISSED** [review]

**Review by lakeside763 — COMMENTED** [review]
There is a conflict to resolve on this PR

**Review by lakeside763 — COMMENTED** [review]
This looks good. But I could not create a peer. because the create button remains disabled after filling the inputs.

File: packages/editor/src/client/route.tsx [hunk 3]
```diff
@@ -122,6 +123,7 @@ export const PeerListRoute = route(RouteType.PeerList, routePath`/peering`)
 export const PeerInfoEditRoute = route(RouteType.PeerProfileEdit, routePath`/peering/profile/edit`)
 export const PeerCreateRoute = route(RouteType.PeerCreate, routePath`/peering/create`)
 export const PeerEditRoute = route(RouteType.PeerEdit, routePath`/peering/edit/${required('id')}`)
+export const MyPeerInfo = route(RouteType.MyPeerInfo, routePath`/peering/peerinfo`)
```
Comment (bishoymelek) [inline packages/editor/src/client/route.tsx:126]: @penk85 do you still need this one?

File: packages/editor/src/client/route.tsx [hunk 4]
```diff
@@ -46,6 +46,7 @@ export enum RouteType {
   PeerProfileEdit = 'peerProfileEdit',
   PeerCreate = 'peerCreate',
   PeerEdit = 'peerEdit',
+  MyPeerInfo = 'myPeerInfo',
```
Comment (bishoymelek) [inline packages/editor/src/client/route.tsx:49]: @penk85 do you still need this one?

**Review by bishoymelek — CHANGES_REQUESTED** [review]

File: packages/editor/src/client/route.tsx (hunk 4 above)
Comment (penk85) [reply packages/editor/src/client/route.tsx:49]: @bishoymelek thanks!

**Review by camilleory — DISMISSED** [review]

**Review by lakeside763 — DISMISSED** [review]

**Review by roosnic1 — APPROVED** [review]

File: packages/editor/src/client/routes/peerList.tsx [hunk 5]
```diff
@@ -38,11 +38,13 @@ import {
   Alert
 } from 'rsuite'
 import {DescriptionList, DescriptionListItem} from '../atoms/descriptionList'
+import {NavigationBar} from '../atoms/navigationBar'
+import {PeerInfoEditPanel} from '../panel/peerProfileEditPanel'
+// import {PeerInfoEditPanel} from '../panel/peerProfileEditPanel'
```
Comment (bishoymelek) [inline packages/editor/src/client/routes/peerList.tsx:43]: do you want to delete this?

**Review by bishoymelek — CHANGES_REQUESTED** [review]

File: packages/editor/src/client/routes/peerList.tsx (hunk 5 above)
Comment (roosnic1) [reply packages/editor/src/client/routes/peerList.tsx]: done

**Review by bishoymelek — APPROVED** [review]
🎉 🎉

**Review by roosnic1 — APPROVED** [review]
🎉
